src/main/java/mate/academy/spring/dao/TicketDao.java

```diff
@@ -1,6 +1,8 @@
 package mate.academy.spring.dao;
 
+import java.util.Optional;
 import mate.academy.spring.model.Ticket;
 
 public interface TicketDao extends GenericDao<Ticket> {
+    Optional<Ticket> get(Long id);
```

Comment:
Why do we need this method?

Reply:
I wantet to use it in OrderController, before I found out from you about "?userId", and didn't know what it means.
I deleted this method in dao and all service layer of ticket